public/locales/de/common.json

```diff
@@ -52,11 +52,11 @@
   "payWithGiroPay": "Spende mit GiroPay",
   "sepaDisclaimer": "SEPA-Zahlungen dauern 2-14 Werktage (in der Regel 3-5 Tage). Sobald sie eingegangen ist, senden wir dir eine Bestätigung per E-Mail mit deinem Baumzertifikat und erhöhen deinen Baumzähler.",
   "sepaMessage": "Indem du deine IBAN angibst und diese Zahlung bestätigst, ermächtigst du Plant-for-the-Planet und Stripe, unseren Zahlungsdienstleister, Anweisungen an deine Bank zu senden, um dein Konto zu belasten. Du hast Anspruch auf eine Rückerstattung durch deine Bank gemäß ihrer allgemeinen Geschäftsbedingungen. Wenn du irgendwelche Probleme mit deiner Spende hast, sende uns bitte eine E-Mail an support@plant-for-the-planet.org.",
-  "bankTransferDisclaimer": "Banküberweisungen benötigen in der Regel 1-5 Werktage, um uns zu erreichen. Sobald deine Überweisung eintrifft, senden wir dir eine Bestätigungs-E-Mail mit deinen Belegen (und deinem Zertifikat).",
+  "bankTransferDisclaimer": "Bitte überweise den Spendenbetrag auf unser Bankkonto. Die Zahlungsdetails zur automatischen Zuordnung erhältst du im nächsten Schritt. Sobald der Zahlungseingang verarbeitet wurde, schicken wir dir eine Bestätigungs-E-Mail, normalerweise dauert dies 2-5 Werktage. Übrigens: Deine SEPA-Überweisungen unterstützt zu 100% das Projekt, da keine Transaktionskosten entstehen.",
```

Comment:
Normally these text are direct translation of the English wording. If we add or change some content, we should think about also adding it to the English text first?

Reply:
Since 96% of the donations via bank transfer come from the DACH region, the German translation seemed more important to me than the English one 😅